src/onboarding/Onboarding.js

```diff
@@ -65,6 +65,7 @@ class Onboarding extends React.PureComponent {
     onResetDaoBuilder: PropTypes.func.isRequired,
     selectorNetworks: PropTypes.array.isRequired,
     visible: PropTypes.bool.isRequired,
+    walletProviderId: PropTypes.string.isRequired,
```

Comment:
💯

Reply:
Yeah, following the flow I saw it was missing :)